prep-javascript-objects/main.js

```diff
@@ -0,0 +1,16 @@
+const person = {
+  firstName: 'Rachael',
+  lastName: 'Kim',
+  hobby: 'eating'
+};
+
+console.log(person);
+
+const fullName = 'Rachael ' + 'Kim';
```

Comment:
Hi Rachael! To get the first and last name of the person object, you can use "dot notation".

```js
var fullName = person.firstName + ' ' + person.lastName;
```